ci/terraform/auth-external-api/shared.tf

```diff
@@ -32,4 +32,7 @@ locals {
   redis_ssm_parameter_policy                  = data.terraform_remote_state.shared.outputs.redis_ssm_parameter_policy
   authentication_oidc_redis_security_group_id = data.terraform_remote_state.shared.outputs.authentication_oidc_redis_security_group_id
   redis_key                                   = "session"
+
```

Comment:
`local.dynatrace_secret` is referenced in `ci/terraform/auth-external-api/token.tf` but not in this list of locals in `ci/terraform/auth-external-api/shared.tf`. Am I missing something?

Reply:
Ok, I think I understand a bit more now. You can reference multiple locals blocks in different terraform files within the same module and they all get merged into one set of locals available across the module.

So in this case, what I see is missing dynatrace_secret is actually in the symlinked ci/terraform/dynatrace.tf file